app/jobs/participant_outcomes/batch_send_latest_outcomes_job.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require Rails.root.join "lib/active_job/queue_adapters/sidekiq_adapter_patch"
+
 module ParticipantOutcomes
   class BatchSendLatestOutcomesJob < ApplicationJob
     queue_as :participant_outcomes
```

Comment:
not introduced in this PR but I realised that the interface for `SendToQualifiedTeachersApiJob` is expecting a keyword parameter, could we change it to:
```
SendToQualifiedTeachersApiJob.perform_later(participant_outcome_id: outcome.id)
```